core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -776,6 +781,42 @@ class UserService(
       } yield {}
     }
 
+  def batchUpdateBillingProjectMembers(projectName: RawlsBillingProjectName,
+                                       batchProjectAccessUpdate: BatchProjectAccessUpdate,
+                                       inviteUsersNotFound: Boolean
+  ): Future[Unit] =
+    requireProjectAction(projectName, SamBillingProjectActions.alterPolicies) {
+      val membersToAdd = batchProjectAccessUpdate.membersToAdd
+      val membersToRemove = batchProjectAccessUpdate.membersToRemove
+
+      collectMissingUsers(membersToAdd.map(_.email), ctx) flatMap { membersToInvite =>
```

Comment:
Nit: I was getting turned around on the naming of `membersToInvite`, perhaps `missingUsers`?

Reply:
Renamed in c3609ba61d3ab4ddc6e065a2208eb655446b6e60